lib/modules/idea-single-widgets/views/widget.html

```diff
@@ -129,8 +129,9 @@ <h4>Deel dit voorstel</h4>
 						<ul>
 							<li><a class="facebook" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u={{data.currentUrl}}">Facebook</a></li>
 							<li><a class="twitter"  target="_blank" href="https://twitter.com/intent/tweet?text={{data.currentUrl}}">Twitter</a></li>
-							<li><a class="email"    target="_blank" href="mailto:?subject={{'Idee voor West: '+idea.title | urlencode}}&body={{data.currentUrl | urlencode}}">Email</a></li>
-							<li><a class="whatsapp" target="_blank" href="https://wa.me/?text={{data.currentUrl | urlencode}}">WhatsApp</a></li>
+							<li><a class="email"    target="_blank" href="mailto:?subject={{'Plan voor ' + data.global.siteTitle + ': '+idea.title | urlencode}}&body={{idea.fullUrl | urlencode}}">Email</a></li>
+							<li><a class="whatsapp desktop" target="_blank" href="https://wa.me/?text={{data.currentUrl | urlencode}}">WhatsApp</a></li>
+							<li><a class="whatsapp mobile" target="_blank" href="whatsapp://send?text={{data.currentUrl | urlencode}}" data-action="share/whatsapp/share">WhatsApp</a></li>
```

Comment:
Kunnen we de `desktop` en `mobile` niet beter op de `li` zetten? Nu laten we een lege li zien

Reply:
Dan moet de styling uiteraard ook op de `li.desktop` en `li.mobile` komen ;)